frontend/src/scenes/onboarding/sdks/sdksLogic.tsx

```diff
@@ -39,10 +41,25 @@ Products that will often be installed in multiple places, eg. web and mobile
 */
 export const multiInstallProducts = [ProductKey.PRODUCT_ANALYTICS, ProductKey.FEATURE_FLAGS]
 
+const getOrderedSDKs = (sdks: SDK[]): SDK[] => {
```

Comment:
This is only "ordered" in terms of product analytics, so I'd be more semantic in this function name

```suggestion
const getOrderedProductAnalyticsSdks = (sdks: SDK[]): SDK[] => {
```